src/plot_api/plot_api.js

```diff
@@ -2124,6 +2126,7 @@ Plotly.update = function update(gd, traceUpdate, layoutUpdate, traces) {
             data: restyleSpecs.eventData,
             layout: relayoutSpecs.eventData
         });
+        gd.emit('plotly_plotmodified');
```

Comment:
Ah, other reason, now that I think about it:

Should afterplot be emitted between frames? The exact sequence definitely matters, so the other logic was that a unique event would free us from tight coupling.